packages/block-library/src/file/edit.js

```diff
@@ -93,6 +99,11 @@ function FileEdit( { attributes, setAttributes, noticeUI, noticeOperations } ) {
 		}
 	}, [] );
 
+	useEffect( () => {
```

Comment:
Whenever the `clientId` changes, the `fileId` changes. That way it's always unique, even if we duplicate the block.